yarn.lock

```diff
@@ -7378,9 +7370,9 @@ script-loader@^0.7.0:
   dependencies:
     raw-loader "~0.5.1"
 
-"scroll-into-view@git+https://github.com/bkirkby/scroll-into-view.git#588b0ced98eeecf84e6fb2074aa076e80b7cffab":
+"scroll-into-view@https://github.com/bkirkby/scroll-into-view.git#588b0ced98eeecf84e6fb2074aa076e80b7cffab":
   version "1.7.4"
-  resolved "git+https://github.com/bkirkby/scroll-into-view.git#588b0ced98eeecf84e6fb2074aa076e80b7cffab"
+  resolved "https://github.com/bkirkby/scroll-into-view.git#588b0ced98eeecf84e6fb2074aa076e80b7cffab"
```

Comment:
I assume these are the same thing? Not sure why this change happened.

Reply:
I'd assume the update was because we're pegged to a minor version and not the patch version